### DISCUSSION TITLE
[Android] Spelling checker has a dark contrast when dark theme is enabled on the OS

### DISCUSSION THREAD

**Discussion by mystic-githuber** [standardnotes/forum#2885] [General]
Spelling checker has a dark contrast when dark theme is enabled on the OS. I am unable to read it properly, this issue doesn't appear when the phone's theme is set to light.

**Comment by effieeee** [comment]
Hi again @mystic-githuber, can I ask for a screenshot of what the spellcheck underline looks like on your end while one of the dark themes are enabled?

**Comment by mystic-githuber** [comment]
@effieeee 

/home/user/Pictures/Screenshot_20221030-180917.png
![Screenshot_20221030-180917](https://user-images.githubusercontent.com/66426048/199126007-574265fc-caa6-4867-b828-f7ae0ce2f609.png)

**Comment by effieeee** [comment]
Thank you for the screenshot! May I also ask what keyboard you're using at the moment, and is your spellcheck menu a unique feature to that keyboard?

**Comment by mystic-githuber** [comment]
@effieeee 

I am using Android AOSP keyboard. The spellchecker is from the standnotes app not the keyboard.
